packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -535,11 +535,26 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
       goldenBytes,
     );
 
+    if (isFlaky) {
+      // TODO(yjbanov): there's no way to communicate warnings to the caller https://github.com/flutter/flutter/issues/91285
+      // ignore: avoid_print
+      print('Golden $golden is marked as flaky and will not fail the test.');
```

Comment:
Can this print statement be within the block of code for when the test fails/flakes? If the test passes, there's no need to add console output.